src/webextension/list/manage/components/account-summary.css

```diff
@@ -124,18 +145,20 @@
   height: 16px;
   background-repeat: no-repeat;
   background-size: 16px 16px;
-  margin-inline-end: 1ch;
+  margin-right: 8px;
```

Comment:
Please continue to use `margin-inline-end` here; it'll look weird otherwise in RTL (granted, the popup will probably end up in the wrong place entirely in RTL, but that's a separate issue).